Fix build for linux >= 4.x

Instead of relying on testing for '3' as the linux kernel major verison,
check that it is 2.x to include the proper Makefile, otherwise forcibly
include the .26 Makefile, that is good for 3.x and 4.x too.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN linux-fusion-9.0.2.orig/Makefile linux-fusion-9.0.2/Makefile
--- linux-fusion-9.0.2.orig/Makefile	2013-07-15 11:58:03.000000000 +0200
+++ linux-fusion-9.0.2/Makefile	2015-06-06 19:03:04.059842652 +0200
@@ -62,10 +62,10 @@
   CPPFLAGS += -DHAVE_LINUX_CONFIG_H
 endif
 
-ifeq ($(K_VERSION),3)
-  KMAKEFILE = Makefile-2.6
-else
+ifeq ($(K_VERSION),2)
   KMAKEFILE = Makefile-2.$(K_PATCHLEVEL)
+else
+  KMAKEFILE = Makefile-2.6
 endif
 
 check-version = $(shell expr \( $(K_VERSION) \* 65536 + $(K_PATCHLEVEL) \* 256 + $(K_SUBLEVEL) \) \>= \( $(1) \* 65536 + $(2) \* 256 + $(3) \))
